packages/zustand-query/src/store.test.ts

```diff
@@ -0,0 +1,155 @@
+import { describe, it, expect } from "vitest";
+import { Table } from "@latticexyz/config";
+import { createStore as createZustandStore } from "zustand/vanilla";
+import { StoreApi } from "zustand";
+import { mutative } from "zustand-mutative";
+import { dynamicAbiTypeToDefaultValue, staticAbiTypeToDefaultValue } from "@latticexyz/schema-type/internal";
+
+/**
+ * A Record is one row of the table. It includes both the key and the value.
+ */
+type Record = { [field: string]: number | string | bigint | number[] | string[] | bigint[] };
```

Comment:
also `boolean` and `boolean[]`

and beware that `Record` conflicts with the TS native `Record`

I think I've used `Row` or `TableRecord` in other areas